app/(main)/o/[slug]/settings/users/user-settings.tsx

```diff
@@ -335,23 +340,25 @@ export default function UserSettings({
                 <DialogHeader>
                   <DialogTitle>Invite users</DialogTitle>
                 </DialogHeader>
-                {effectiveSeats !== undefined && currentPlan !== "developer" && (
-                  <div className="text-sm text-[#74747A] mt-2">
-                    {(() => {
-                      const currentUsage =
-                        Number(totalUsers) + Number(totalInvites) + (form.getValues("emails")?.length || 0);
-                      const remaining = effectiveSeats - currentUsage;
-                      if (remaining > 0) {
-                        return `${remaining} seat${remaining === 1 ? "" : "s"} left`;
-                      } else if (remaining === 0) {
-                        return "0 seats left";
-                      } else {
-                        const additional = Math.abs(remaining);
-                        return `0 seats left, ${additional} additional seat${additional === 1 ? "" : "s"} required`;
-                      }
-                    })()}
-                  </div>
-                )}
+                {effectiveSeats !== undefined &&
```

Comment:
This is fine, but I think we're at the point where we should consider splitting it out into its own component. If it gets any more complicated, definitely.